scripts/xe-getarrayidentifier

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/python
+#!/usr/bin/python3
```

Comment:
This entire script is obsolete, remove it, it was only used by the MPP multipath support which was removed some time ago.